content/inside-rust/program-management-update-2025-08.md

```diff
@@ -0,0 +1,270 @@
+
+
+## Content Team
+
+This month also saw a formation of a [new team](https://www.rust-lang.org/governance/teams/launching-pad#team-content) focused on producing audio/video/text content about Rust and people working on it. These can be interviews, podcasts etc.
+
+[TC](https://github.com/traviscross) and [Pete LeVasseur](https://github.com/PLeVasseur) are the leads, and we also have [Cameron Dershem](https://github.com/cldershem), [Xander Cesari](https://github.com/MerrimanInd), [Tyler Mandry](https://github.com/tmandry), [Lori](https://github.com/LoriLorusso), and yours truly.
+
+[rustconf]: https://rustconf.com/
+
+We already have a few interviews planned for [RustConf 2025][rustconf].
+
+Here's the [Content Team's charter](https://github.com/rust-lang/leadership-council/issues/206). Forming a new team is something you propose to the Leadership Council in their repo's issues. It then gets discussed at their meeting and decided on using the FCP process.
+
+
+## build-std
+
+[build-std](https://rust-lang.github.io/rust-project-goals/2025h2/build-std.html) is an ongoing initiative to provide a blessed, stable process for building the Rust standard library you can use instead of the one we provide.
+
+There are many different motivations for this, for example supporting platforms where Rust doesn't ship a precompiled library, optimizing it to known hardware or reduce its size (by e.g. removing features that are not necessary). This is of big interest to the [Rust for Linux project](https://rust-for-linux.com/).
+
+[David Wood](https://github.com/davidtwco) and [Adam Gemmell](https://github.com/adamgemmell) wrote a comprehensive document describing its history, motivations, past experiments, and proposals for an minimal scope that they can start building.
+
+This has been regularly reviewed by a handful of people across the teams that will be affected by this. After many rounds of feedback, David feels the proposal is solid enough to open to a broader group.
+
+He's shared it with more representatives from the Libs, crates.io, embedded, bootstrap and infra and compiler and Cargo teams. He's also shared it with non-Cargo users e.g. the Rust for Linux folks (who are interested in building std without Cargo).
+
+Once this next round settles down, he will open the RFC (it will likely be several documents each focusing on a different stage of the effort).
+
+## Rust for Linux
+
+[Rust for Linux](https://rust-for-linux.com/) is an ongoing initiative to be able to write Linux kernel code in Rust. The motivation is memory safety, a modern language and bring in more contributors due to the combination of the two.
+
+The focus in this second half of 2025 is to bring unstable features the project is using into stable Rust. Linux takes stability and backwards compatibility very seriously and will not rely on features that can change from release to release or even be removed entirely.
+
+### RFC: pass pointers to `const`
```

Comment:
@Darksonn I'm trying to provide more context in the PM updates (so people know why we're the things mentioned here are important etc.). If you have a bit of time, would you please check whether this is accurate and possibly suggest any improvements?